app/services/search/feed_content.rb

```diff
@@ -44,6 +44,7 @@ def prepare_doc(hit)
           "image_url" => source["main_image"],
           "title" => source["title"]
         }
+        source["_score"] = hit["_score"]
```

Comment:
Added this return value mainly for debugging purposes and additional information that is nice to have. It is not actually going to be used by anything in the code.